packages/appeals-service-api/src/routes/v2/documents/spec.yaml

```diff
@@ -0,0 +1,21 @@
+paths:
+  /api/v2/service-users/:
```

Comment:
docs here and further down

Reply:
they should just take my c and v keys away probably